scriptmodules/libretrocores/lr-snes9x.sh

```diff
@@ -41,6 +41,8 @@ function configure_lr-snes9x() {
     mkRomDir "snes"
     ensureSystemretroconfig "snes"
 
+    local def=0
+    ! isPlatform "armv6" && ! isPlatform "armv7" && def=1
     addEmulator 0 "$md_id" "snes" "$md_inst/snes9x_libretro.so"
```

Comment:
does this want to be `addEmulator $def` rather than `addEmulator 0`?

Reply:
It does yep.